src/Program.fs

```diff
@@ -2828,7 +2851,7 @@ let runConfig filePath =
                     if config.target = Target.FSharp then
                         XElement.PackageReference("Fable.Remoting.Json", "2.18.0")
                         XElement.PackageReference("Newtonsoft.Json", "13.0.1")
-                        if config.asyncReturnType = AsyncReturnType.Task
+                        if config.IsTask
```

Comment:
I believe atm we target netstandard2.0, if we target `net6.0` we should be able to remove `Ply` dependency but that's for another work item

Reply:
??
```
  <PropertyGroup>
    <Description>dotnet CLI tool to generate type-safe F# clients from OpenAPI/Swagger services</Description>
    <OutputType>Exe</OutputType>
    <ToolCommandName>hawaii</ToolCommandName>
    <TargetFramework>net6.0</TargetFramework>
    <PackAsTool>true</PackAsTool>
    <IsPackable>true</IsPackable>
    <RollForward>Major</RollForward>
    <Version>0.63.0</Version>
    <PackageReleaseNotes>Handle reference payload on return types</PackageReleaseNotes>
  </PropertyGroup>

```

Reply:
I meant the generated client, not the CLI itself